scripts/generate-icon-font.ts

```diff
@@ -40,14 +41,19 @@ async function main(): Promise<void> {
   });
 
   // Prints out the VSCode configuration package.json
-  console.info(
-    JSON.stringify(
-      {
-        icons: iconsConfig,
-      },
-      undefined,
-      2
-    )
+  const currentConfiguration = JSON.parse(
+    await fs.readFile('./package.json', 'utf8')
+  );
+
+  currentConfiguration.contributes.icons = iconsConfig;
+
+  const prettierConfig = await prettier.resolveConfig('./.prettierrc.json');
+  await fs.writeFile(
+    './package.json',
+    prettier.format(JSON.stringify(currentConfiguration), {
+      ...prettierConfig,
+      parser: 'json-stringify',
```

Comment:
Which exactly rules are applied here? I am wondering what is there on top of `JSON.stringify(..., null, 2)`.

Reply:
I think this does end up being equivalent to that, it's just tied to [our prettier config](https://github.com/mongodb-js/devtools-shared/blob/d9db8b5b49c745d29d17097f439be14d565092c5/configs/prettier-config-devtools/index.json) (and we have nearly nothing there so... this should be default prettier) so it just stays consistent with any style rules for our `prettier` formatter. This way if someone autoformats the file using VSCode there won't be inconsistencies.